R/common-find-trends.R

```diff
@@ -0,0 +1,94 @@
+#' @title
+#' Find any columns that have a trend above a particular threshold
+#' @description
+#' Find numeric columns in data frame that have an absolute slope greater than
+#' that specified via threshold argument.
```

Comment:
The example for this function could be a little clearer. Am I setting a threshold somewhere? Comparing two columns? How? Absolute slope of what?